src/components/BorrowerProfile/LendCta.vue

```diff
@@ -430,10 +431,10 @@ export default {
 			);
 		},
 		cycleStatsSlot() {
-			if (this.matchingText.length && !isMatchAtRisk) {
+			if (this.matchingText.length) {
 				const cycleSlotMachine = () => {
-					if (this.statScrollAnimation) {
-						this.statScrollAnimation = false;
+					if (!this.isMatchAtRisk) {
+						this.statScrollAnimation = !this.statScrollAnimation;
```

Comment:
This covers the previous if/else condition and we only enable this swapping if the loan is matching and not at risk.